netbeans.apache.org/src/content/templates/footer.gsp

```diff
@@ -94,11 +97,8 @@
         <p>Licensed under the <a href="//www.apache.org/licenses/">Apache Software License, version 2.0.</a></p>
```

Comment:
Missing "https:"?

Reply:
The URL starting with "//" mean: Use what ever protocol is currently used to access the referenced resource. So if the page was loaded via "http" also access apache.org via http and if the page was loaded via "https", access apache.org via "https". So it could be intentional.

Reply:
I see. But License Type is red now. So should we write it explicitly?

Reply:
Thanks for this, @junichi11 and @matthiasblaesing . The '//' was indeed intentional, but it's better to make 'https' explicit.